tests/ui/borrowck/borrowck-vec-pattern-nesting.rs

```diff
@@ -88,6 +90,7 @@ fn e() {
     //~| NOTE cannot move out of here
     //~| NOTE move occurs because
     //~| HELP consider borrowing here
+    //~| NOTE in this expansion of desugaring of a resized `Span`
```

Comment:
this note shows up in json but not in rendered diagnostics

Reply:
I don't think we'll ever want it to come up. I'll see if I can fully silence it.

Reply:
I had some trouble finding the right place to silence this. I thought `emitter.rs` was the right place, but the place where this seemed to be coming from, wasn't.

Reply:
🤷 not really a problem